test/tests/functional/pbs_reservations.py

```diff
@@ -2270,3 +2270,45 @@ def test_resv_job_hard_walltime(self):
         J = Job(TEST_USER, attrs=a)
         jid = self.server.submit(J)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+    def test_resv_reconfirm_holding_partial_nodes(self):
+        """
+        Test that scheduler is able to reconfirm a reservation when
+        only of the nodes reservation was running on goes down. Also
+        make sure it hangs on to the node that was not down.
+        """
+        a = {'reserve_retry_time': 5}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+        a = {'resources_available.ncpus': 2}
+        self.server.create_vnodes('vn', a, num=3, mom=self.mom)
+        vn_list = ['vn[0]', 'vn[1]', 'vn[2]']
+
+        now = int(time.time())
+        sel = '1:ncpus=2+1:ncpus=1'
+        rid = self.submit_reservation(user=TEST_USER, select=sel,
+                                      start=now + 5, end=now + 300)
+
```

Comment:
If you want to truly follow the pep8 standard, it says to put spaces around your operators except in function calls.  It doesn't really matter though.